cordapp-cpk2/src/test/kotlin/net/corda/plugins/cpk2/json/CPKDependenciesReader.kt

```diff
@@ -0,0 +1,11 @@
+package net.corda.plugins.cpk2.json
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import java.io.InputStream
+
+class CPKDependenciesReader{
+    companion object {
+        fun loadCPKDependencies(input: InputStream): List<CPKDependency> =
+            jacksonObjectMapper().readValue(input, CPKDependencyFile::class.java).dependencies.toList()
```

Comment:
You can move this function into `CPKDependency.kt` rather than creating a class only to contain a `companion object`.

This also looks like it creates a new `ObjectMapper` every time it is invoked, which sounds expensive. I think you can create an `ObjectMapper` up-front, and then reuse it:
```kotlin
import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
import com.fasterxml.jackson.module.kotlin.readValue

private val mapper = jacksonObjectMapper()

fun loadCPKDependencies(input: InputStream): List<CPKDependency> =
    mapper.readValue<CPKDependenctFile>(input).dependencies
```

Reply:
refactored